pulseguardian/guardian.py

```diff
@@ -43,10 +43,29 @@ def __init__(self, api, emails=True, warn_queue_size=config.warn_queue_size,
         self.emails = emails
         self.warn_queue_size = warn_queue_size
         self.del_queue_size = del_queue_size
+        self._polling_interval = config.polling_interval
+
+        self._connection_error_notified = False
+        self._unknown_error_notified = False
```

Comment:
Since these are internal attributes, it would be better to put them after all the others that are passed in via arguments (so down after self.on_delete). Also I don't know why line 46 is blank right now, so you can remove that at the same time. :)